tracks/ruby/exercises/acronym/mentoring.md

```diff
@@ -22,11 +22,23 @@ with variations for the regex (see below).
 In all these cases: it's recommended to suggest the `String#scan` method and, because the exercise is not about Regular Expressions, give them also the RegEx (`/\b[a-zA-Z]/`). (At this point in the track, it's more about experiencing the strengths of Ruby's built-in methods than making them find them. There's ample opportunity for research later on in the track.)   
 
 ## Talking points
-* If the solution uses Class instead of Module: that's totally acceptable at this stage. No reason to discuss.
+* `Class vz Module`: If the solution uses Class instead of Module: that's totally acceptable at this stage. No reason to discuss.
```

Comment:
Not sure if this was a typo!

```suggestion
* `Class vs Module`: If the solution uses Class instead of Module: that's totally acceptable at this stage. No reason to discuss.
```

Reply:
Thanks :-)